src/main/java/dev/tycho/stonks/command/company/ServicesCommandSub.java

```diff
@@ -16,10 +16,15 @@
 
   @Override
   public void onCommand(Player player, String alias, String[] args) {
-    if (args.length > 1) {
-      DatabaseHelper.getInstance().openCompanyServices(player, concatArgs(1, args));
-    } else {
-      player.sendMessage(ChatColor.RED + "Please specify a company!");
+    if (args.length < 2){
+      sendMessage(player, ChatColor.RED + "Please specify a company!");
+      return;
     }
+    if (!validateDouble(args[1])) {
```

Comment:
What? is this a company name or a number?

If Number: Use `StringUtils#isNumeric`